save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/ProposeYourContest.kt

```diff
@@ -19,57 +19,44 @@ import kotlinx.js.jso
 @Suppress("MAGIC_NUMBER")
 fun ChildrenBuilder.proposeContest() {
     div {
-        className = ClassName("col-lg-3")
+        className = ClassName("row mt-3")
+
         div {
-            className = ClassName("card flex-md-row mb-1 box-shadow")
-            style = jso {
-                minHeight = 10.rem
+            className = ClassName("col-lg-3 pr-0")
+            img {
+                src = "img/undraw_mailbox_re_dvds.svg"
+                style = jso {
+                    width = "100%".unsafeCast<Width>()
+                }
             }
+        }
 
-            div {
-                className = ClassName("col-lg-12 mt-2")
-                div {
-                    className = ClassName("row")
-
-                    div {
-                        className = ClassName("col-lg-3 pr-0")
-                        img {
-                            src = "img/undraw_mailbox_re_dvds.svg"
-                            style = jso {
-                                width = "100%".unsafeCast<Width>()
-                            }
-                        }
-                    }
-
-                    div {
-                        className = ClassName("col-lg-9")
-                        p {
-                            +"Want to make your own contest? Write us an e-mail:"
-                        }
-                    }
-                }
+        div {
+            className = ClassName("col-lg-9")
+            p {
+                +"Want to make your own contest? Write us an e-mail:"
+            }
+        }
+    }
 
-                div {
-                    className = ClassName("row")
-                    div {
-                        className = ClassName("col-lg-3 pr-0")
-                    }
+    div {
+        className = ClassName("row")
+        div {
+            className = ClassName("col-lg-3 pr-0")
+        }
 
-                    div {
-                        className = ClassName("col-lg-9")
-                        style = jso {
-                            justifyContent = JustifyContent.center
-                            display = Display.flex
-                            alignItems = AlignItems.center
-                            alignSelf = AlignSelf.flexEnd
-                        }
+        div {
+            className = ClassName("col-lg-9")
+            style = jso {
+                justifyContent = JustifyContent.center
+                display = Display.flex
+                alignItems = AlignItems.center
+                alignSelf = AlignSelf.flexEnd
+            }
 
-                        strong {
-                            className = ClassName("d-inline-block mb-2 text-success")
-                            +"saveourtool@gmail.com"
-                        }
-                    }
-                }
+            strong {
+                className = ClassName("d-inline-block mb-2 text-success")
+                +"saveourtool@gmail.com"
```

Comment:
maybe make it clickable?
Like

```suggestion
            a {
                href = "mailto:saveourtool@gmail.com"
                +"saveourtool@gmail.com"
```

Reply:
very good idea